pagopa/io-app
# feat: [IOCOM-528] Add check on minimum app version for PN

## Short description
This PR introduces the check on the minimum app version for PN. If the app version is lower than the minimum required version, the user must update the app to use the functionality.
There are two entry points:
- precondition bottom sheet
- special services cta

**_Note_**: the check in the precondition bottom sheet is specific to PN. In the future it could be generic.

## List of changes proposed in this pull request
- removed local FF
- updated `SpecialServicesCTA.tsx`: the button to update the app is displayed if the app version is lower than the minimum required version
- updated `useMessageOpening.tsx`: a blocking message is displayed to allow the user to update the app
- updated io-services-metadata definitions
- updated translations

## How to test
Using io-dev-api-server, increment the `min_app_version` for PN (e.g. 3.35.0.1) and verify that:
- the button to update the app is visible in the service
- the precondition bottom sheet shows a blocking message
